src/components/SearchSection.js

```diff
@@ -87,6 +87,8 @@ const SearchSection = (props) => {
         return wordInformation["wordform_text"][displayType]
     }
 
+    console.log("PROPS IN SEARCH SECTION");
+    console.log(props);
```

Comment:
Consider removing these in a future PR. I can see the benefit of knowing what's in here for now, though, so no worries about it this time.